mlir/tools/imex-opt/passes.cpp

```diff
@@ -26,6 +26,8 @@
 #include "mlir-extensions/Conversion/gpu_to_gpu_runtime.hpp"
 #include "mlir/Dialect/GPU/Transforms/Passes.h"
 
+// Passes registration.
+
```

Comment:
LLVM convention is to do something like this at the top of the file:

```
//===- passes.cpp - IMEX passes ---------------------------------*- C++ -*-===//
//
// Copyright 2022 Intel Corporation
// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
///
/// \file This file registers all IMEX passes by instantiating static variables.
///
//===----------------------------------------------------------------------===//
```

Reply:
We are using different convention in the main currently and it is better change it entire project in single PR instead of changing random files in random reviews.

Reply:
For real?